vector/v.qcount/v.qcount.html

```diff
@@ -127,8 +127,3 @@ <h2>AUTHORS</h2>
 <p>Modified for GRASS 5.0 by Eric G. Miller (2000-10-28)
 <br>
 Modified for GRASS 5.7 by R. Blazek (2004-10-14)
-
-<!--
-<p>
-<i>Last changed: $Date$</i>
--->
```

Comment:
`<h2>REFERENCES</h2>` and `<h2>KNOWN ISSUES</h2>` section should be before `<h2>SEE ALSO</h2>` section.

Line 82 in `<h2>REFERENCES</h2>`  section  unnecessary one empty line.